src/transformers/modeling_reformer.py

```diff
@@ -100,6 +114,23 @@ def _get_least_common_mult_chunk_len(config):
         )
 
 
+def _get_min_chunk_len(config):
+    attn_types = config.attn_layers
+    attn_types_set = set(attn_types)
+    if len(attn_types_set) == 1 and attn_types[0] == "lsh":
```

Comment:
(NITCITY)
you can do set equality I think. You can also type hint instead of including type in varname.
```python
if attn_types == {'lsh'}:
```